commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java

```diff
@@ -21,8 +29,18 @@ public DlpPositiveItemServiceImpl(DlpPositiveItemDAO dlpPositiveItemDAO) {
 
 
     @Override
-    public List<DlpPositiveItemEntity> getDlpPositivesItems() {
-        return dlpPositiveItemDAO.findAll();
+    public DlpPositiveItemList getDlpPositivesItems(int offset, int limit) throws Exception {
+        DlpPositiveItemList dlpPositiveItemList = new DlpPositiveItemList();
+        List<DlpPositiveItemEntity> dlpPositiveItemEntities = dlpPositiveItemDAO.findAll();
```

Comment:
Here you should not make a find all. 
Use offset and limit to make a sql query containing a limit option

Reply:
You can inspire by https://github.com/exoplatform/app-center/blob/develop/app-center-services/src/main/java/org/exoplatform/appcenter/dao/ApplicationDAO.java#L39

Reply:
Done!